src/test/resources/environment.yaml

```diff
@@ -11,49 +12,135 @@ proxy-config:
+global-tags:
+  # Name: cerberus-{env} is automatically added to this map
+  cost-center: 11111
+  owner-email: obvisouly.fake@nike.com
+  owner-group: engineering-team-name
 
 # A IAM role ARN that will be given elevated privileges for the KMS CMK created.,
-# If you don't separate root access from admins just use the root role here
-admin-role-arn: arn:aws:iam::111111111:role/onelogin-roles-OneLoginAdminRole-2222222222
-# The Route 53 hosted zone name that will be created for CNAME records used by internal ELBs.
-vpc-hosted-zone-name: demo.internal.cerberus-oss.io
-# The hostname that will be exposed for Cerberus
-hostname: demo.cerberis-oss.io
-# The hostedZoneId that will allow the registration of the hostname
+# If you don't separate root access from admins just use the root role here.
+admin-role-arn: arn:aws:iam::111111111:role/admin
+
+# This is used to create dns records and certificates
+# By default the following records will be created
+# {environment-name}.{base-domain} ex: demo.example.com
+# origin.{environment-name}.{base-domain} ex: origin.demo.example.com
+# {environment-name}.{primary-primaryRegion}.{base-domain} ex: demo.us-west-2.example.com
+# In this example demo.example.com will point to origin.demo.example.com which points to demo.us-west-2.example.com which points to the ALB
+# This extra hop allows for more flexibility in future infrastructure operations
+base-domain-name: example.com
+
+# Edge domain name defaults to {environment}.{base-domain}
+edge-domain-name-override: highlander-demo.example.com
+# Origin domain name defaults to origin.{environment-name}.{base-domain} override it here
+origin-domain-name-override: origin.highlander-demo.demo.com
+# Load balancer domain name defaults to {environment-name}.{primary-primaryRegion}.{base-domain} override it here
+load-balancer-domain-name-override: highlander-demo.us-west-2.demo.com
+
+# Add additional subject names to the generated cert
+# For example if your standing up a new highlander edition of Cerberus and you still have the existing env up you can add those domains
+# here so your certificates are still valid when you cut over the edge domain name to the old domain name.
+additional-subject-names:
+  - demo.example.com
+  - origin.demo.example.com
+  - demo.us-west-2.com
+  - demo.us-east-1.com
+
+# The SSL Policy that will get applied to the application load balancer
+# see http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html
+# If this is not defined than we default to 'TLS-1-2-2017-01' in the old cerberus arch we defaulted to
+# 'ELBSecurityPolicy-2016-08' if you have clients running on deprecated Java versions and need to maintain tls 1.0 and
+# 1.1 support use ELBSecurityPolicy-2016-08
+# load-balancer-ssl-policy-override: ELBSecurityPolicy-2016-08
+
+# The hosted zone id that will allow the registration of the above domain names
 hosted-zone-id: X5CT6JROG9F2DR
```

Comment:
Do we want a hosted-zone-id checked in?

Reply:
Is this a public AWS zone or one of ours?

Reply:
Thats a fake hosted zone